### PR TITLE
Remove TokenState::Retired event, which is not reliable on Multi-Mon going forward

### PR DESCRIPTION
**Pull request by Zu-shi**
TokenState::Retired events are no longer guaranteed to be sent at the end of a frame, so we use DWM's present stats to understand the Composed Flip timeline instead.

Signed-off-by: Zuoming Shi <zushi@microsoft.com>

### CODE REVIEW

File: PresentData/PresentMonTraceConsumer.cpp [hunk 1]
```diff
@@ -1300,9 +1303,11 @@ void PMTraceConsumer::HandleDWMEvent(EVENT_RECORD* pEventRecord)
         if (eventIter != mWin32KPresentHistoryTokens.end()) {
             TRACK_PRESENT_PATH(eventIter->second);
             DebugModifyPresent(*eventIter->second);
-            eventIter->second->DwmNotified = true;
```
Comment (jenatali) [inline PresentData/PresentMonTraceConsumer.cpp:1310]: Why'd you delete the setting of `DwmNotified`?

File: PresentData/PresentMonTraceConsumer.cpp [hunk 2]
```diff
@@ -1161,6 +1162,8 @@ void PMTraceConsumer::HandleWin32kEvent(EVENT_RECORD* pEventRecord)
         {
             TRACK_PRESENT_PATH(eventIter->second);
 
+            event.SeenInFrameEvent = true;
```
Comment (jenatali) [inline PresentData/PresentMonTraceConsumer.cpp:1165]: I thought the ordering here was `ScheduleCompositionSurfaceUpdate` first, and then `InFrame` afterwards. Is it inverted?

Comment (Zu-shi) [reply PresentData/PresentMonTraceConsumer.cpp:1165]: Yeah, InFrame comes first. From test_case_1.etl
image

(A second image to fit the Composition Surface Luid in)
image

File: PresentData/PresentMonTraceConsumer.cpp (hunk 1 above)
Comment (Zu-shi) [reply PresentData/PresentMonTraceConsumer.cpp:1310]: Ah, that was a typo, fixed.

File: PresentData/PresentMonTraceConsumer.cpp (hunk 2 above)
Comment (jenatali) [reply PresentData/PresentMonTraceConsumer.cpp:1165]: Huh, fair enough.

**Review by jenatali — APPROVED** [review]
Looks okay to me now.

**Review by jenatali — APPROVED** [review]
Makes sense to me.

**Pull request merged**